FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/presenter/LoginPresenter.java

```diff
@@ -0,0 +1,4 @@
+package com.pps.globant.fittracker.mvp.presenter;
+
+public class LoginPresenter {
```

Comment:
Please instantiate this presenter with s simple activity with a place holder text